pages/dkp/kaptain/2.0.0/install/index.md

```diff
@@ -10,6 +10,8 @@ enterprise: false
 
 <p class="message--important"><strong>IMPORTANT: </strong>Refer to the <a href="../fresh-install">Kaptain fresh install instructions</a>, if you already have Kaptain and want to move from 1.x to 2.x. Note that it is not possible to migrate your data.</p>
 
+<p class="message--important"><strong>IMPORTANT: </strong>Ensure the cluster that you want to use to deploy Kaptain is the only cluster in its workspace. **Kaptain is meant to be deployed on workspaces with a single cluster**.</p>
```

Comment:
this warning is at the bottom of [the corresponding page on confluence](https://d2iq.atlassian.net/wiki/spaces/DKAP/pages/13271060/Install+Kaptain). not sure which one's better but we could try being consistent

Reply:
100%! Changed to the top in Confluence.